packages/app-store/routing-forms/pages/routing-link/getUrlSearchParamsToForward.ts

```diff
@@ -80,15 +81,34 @@ export function getUrlSearchParamsToForward({
     }
   }
 
+  const attributeRoutingConfigParams: Record<string, any> = {};
+
+  if (attributeRoutingConfig) {
+    for (const key of Object.keys(attributeRoutingConfig)) {
+      if (key === "skipContactOwner" && attributeRoutingConfig[key]) {
+        attributeRoutingConfigParams["cal.skipContactOwner"] = "true";
+      }
+
+      // TODO: How do we move this logic to their respective app packages
+      if (key === "salesforce") {
+        const salesforceData = attributeRoutingConfig[key];
+
+        if (salesforceData?.rrSkipToAccountLookupField && salesforceData.rrSKipToAccountLookupFieldName) {
+          attributeRoutingConfigParams["cal.salesforce.rrSkipToAccountLookupField"] = "true";
```

Comment:
Only want to send to the URL that this option is enabled. Don't want to send the specific field name as that is unique to each Salesforce instance.